### DISCUSSION TITLE
401 - when i am trying to install zed

### DISCUSSION THREAD

**Discussion by sylvainmouquet** [zed-industries/zed#10010] [Feature Requests]
Hello,

I have tried to install zed but i cannot i have error 401
`brew install zed`

`Error: Download failed on Cask 'zed' with message: Download failed: https://zed.dev/api/releases/stable/0.128.3/Zed-x86_64.dmg`

**Comment by Moshyfawn** [comment]
The Zed team doesn't maintain the Homebrew version; it's a community cask.

**Comment by petros** [comment]
Could you try a `brew update` before trying again?

**Comment by sylvainmouquet** [comment]
It's better now, i had also the error when i click on download links in https://zed.dev/releases/stable
but it's solved now, i close the discussion
